include/glow/Base/Type.h

```diff
@@ -113,6 +113,31 @@ struct ShapeNCHW {
   }
 };
 
+struct ShapeTLBR {
+  size_t pad_t; // top padding
```

Comment:
I think that `pad_` part is redundant here. You can use it like this:
```
ShapeTLBR pads = ...;
...
x = fx - pads.t;
y = fy - pads.l;
```
Maybe it's even better to extend t to top, l to left, etc.

I don't think that we'll use `ShapeTLBR` for anything else, because kernel and stride are more like `ShapeHW`, there's only 1 number per axis.